### PR TITLE
Blob Overminds can now ghost.

### LINKED ISSUE #23549: Blob overminds can't ghost or suicide
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1620

### Issue Description

You cannot ghost or suicide as a blob overmind

### What did you expect to happen?

To be able to at least ghost out of the blob

### What happened instead?

Stuck in blob forever, can't even kill myself to ghost

### Why is this bad/What are the consequences?

At roundend, it can be nice to be able to ghost to see what happened to the rest of the crew/station. For example, just now, the shuttle got successfully hijacked whilst I was playing blob, with my split consciousness overmind being abroad. I really wanted to see how that happened, but couldn't ghost to go see it. 

### Steps to reproduce the issue.

Play blob, try to ghost or suicide

### When did the problem start happening?

No clue

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by PIerreLouisH**
## What Does This PR Do
Blob Overminds can now ghost. Therefore Fixes #23549

## Why It's Good For The Game
Oversight fix gud

## Images of changes
image

## Changelog
:cl:
fix: Blob Overminds can now ghost.
/:cl:

### CODE REVIEW

File: code/modules/events/blob/overmind.dm [hunk 1]
```diff
@@ -122,3 +122,23 @@
...
+	if(alert("Are you -sure- you want to ghost? (You are alive. If you ghost now, you probably won't be able to rejoin the round! You can't change your mind, so choose wisely!)", "Confirm Ghost", "Yes", "No") != "Yes")
+		return
+
+	// Create a ghost from the blob overmind
+	var/mob/dead/observer/ghost = new(get_turf(src))
+	ghost.ckey = src.ckey
+	ghost.name = src.name
+
+	// Clear the blob overmind's key to disconnect the player
+	src.ckey = null
```
Comment (Contrabang) [inline code/modules/events/blob/overmind.dm:141]:
```suggestion
	ghostize(FALSE)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
making observers shouldnt be done manually, use ghostize instead

**Review by lewcc — CHANGES_REQUESTED** [review]
The main `ghost()` verb is scoped under /mob/living, and this skips a few checks that we would probably like to be sure are accounted for (like respawnability). 

I think it'd be better if you were move the existing ghost verb up to `/mob`, and add overrides for the different mob types that wouldn't use the base type (such as `/mob/dead`, and `/mob/camera/aiEye`)?

**Review by S34NW — CHANGES_REQUESTED** [review]
PR is stale, please continue working on it or address any comments within 7 days or we will close the PR.

**Pull request closed without merging**